src/main/java/io/kuzzle/sdk/security/KuzzleSecurity.java

```diff
@@ -361,7 +361,7 @@ public void updateRole(@NonNull final String id, final JSONObject content, final
         @Override
         public void onSuccess(JSONObject response) {
           try {
-            listener.onSuccess(response.getJSONObject("result").getString("_id"));
+            listener.onSuccess(new KuzzleRole(KuzzleSecurity.this.kuzzle, response.getJSONObject("result").getString("_id"), response.getJSONObject("result")));
```

Comment:
It's a document error then. Here is the actual `updateRole` response from Kuzzle:

```json
{
  "action": "updateRole",
  "controller": "security",
  "error": null,
  "metadata": {},
  "requestId": "724f8361-0e02-4918-8f6a-2c6c0406adb3",
  "result": {
    "_id": "foo",
    "_source": {
      "controllers": {
        "*": {
          "actions": {
            "*": true
          }
        }
      }
    }
  },
  "status": 200
}
```